pkgs/by-name/lo/local-ai/package.nix

```diff
@@ -147,8 +147,8 @@ let
         src = fetchFromGitHub {
           owner = "ggerganov";
           repo = "llama.cpp";
-          rev = "fc54ef0d1c138133a01933296d50a36a1ab64735";
-          hash = "sha256-o87EhrA2Oa98pwyb6GSUgwERY0/GWJiX7kvlxDv4zb4=";
+          rev = "70392f1f81470607ba3afef04aa56c9f65587664";
+          hash = "sha256-PgdH3Q4lM45a7aLeN/19i0Ld4NuTfzh/caYOSdooFao=";
           fetchSubmodules = true;
         };
```

Comment:
Question: can't we use the existing `llama.cpp` derivation ?

Reply:
The coupling between local-ai and llama.cpp is on C++ level. I think it is simpler to maintain the packe with the upstream used llama.cpp version.